2048_fib_mini project/moves_handling_functions.c

```diff
@@ -0,0 +1,192 @@
+//moves_handling_functions.c
+#include<stdio.h>
+#include"fibonacci_checkers.c"
+//functions declaration
+
+void downMove(int *,int );
```

Comment:
move all declarations to `*.h` files and keep implementations in `*.c/cpp` files. Not just for this all files where you divided the code.

Reply:
ohk this is done